samcli/lib/utils/managed_cloudformation_stack.py

```diff
@@ -59,11 +59,10 @@ def _create_or_get_stack(cloudformation_client, stack_name, template_body):
         ds_resp = cloudformation_client.describe_stacks(StackName=stack_name)
         stacks = ds_resp["Stacks"]
         stack = stacks[0]
-        click.echo("\n\tLooking for resources needed for deployment: Found!")
         _check_sanity_of_stack(stack, stack_name)
         return stack["Outputs"]
     except ClientError:
-        click.echo("\n\tLooking for resources needed for deployment: Not found.")
+        pass
```

Comment:
Is this something we still need to have? or do we have additional statements which already reflect it and it was a duplicate.

Reply:
This is an UX change that moves "Found"/"Not Found" to the individual managed resources.
```
Looking for resources needed for deployment:
 Managed S3 bucket: aws-sam-cli-managed-default-samclisourcebucket
 Image repositories: Not found.
```

Reply:
We should maybe have a log.debug here as well.